client/state/analytics/README_LuckyOrange.md

```diff
@@ -0,0 +1,17 @@
+Lucky Orange Analytics
+========
+
+Lucky Orange is an analytics platform that offers a suite of tools that helps you understand your users better. It's a research tool that offers insight into visitor behaviour. Out of the box, all data sent to Lucky Orange is anonymous. It does not capture keystroke data. Every character from every field on every page is replaced with an asterisk before any data is recorded. This is to keep all data anonymous.
```

Comment:
yay! a README!

forgive me for my ignorance, but I still don't know from this paragraph what it does or why we have it. I know what it supposedly _doesn't_ do but not what it _does_ do.

does it track every mouse movement? use the webcam to record emotional responses? a little explanation could be incredibly helpful!